e2e/generateDeployments.js

```diff
@@ -179,8 +180,8 @@ async function run() {
   log('Writing', `deployments/meta.json`);
   await fs.writeFile(`${__dirname}/deployments/meta.json`, JSON.stringify(meta, null, 2));
 
-  log('Writing', `deployments/extras.json`);
-  await fs.writeFile(`${__dirname}/deployments/extras.json`, JSON.stringify(extras, null, 2));
+  log('Writing', `deployments/settings.json`);
```

Comment:
This is unnecessary change. There was no need to rename extras to settings and make breaking changes in libraries I support. I will revert all the rest.